app/assets/javascripts/discourse/app/lib/autocomplete.js

```diff
@@ -117,6 +117,26 @@ export default function (options) {
     discourseLater(() => me.trigger("keydown"), 50);
   }
 
+  function scrollAutocomplete() {
+    if (!div) {
+      return;
+    }
+
+    const divEl = div[0];
```

Comment:
I guess `El` stands for element? Not sure we should use it since it's not a commonly used abbreviation.

Reply:
Really? I’ve seen it used and used it tons over the years. But will change to element for clarity :)

Reply:
Yeah, I don't recall seeing it a lot, though I checked now and there are some instances in Discourse Core.